tests/main/snap-connect/task.yaml

```diff
@@ -30,4 +30,24 @@ execute: |
 
     echo "The plug can be connected to a slot on the core snap using abbreviated syntax"
     snap connect home-consumer:home :home
+
     snap interfaces | MATCH "$CONNECTED_PATTERN"
+
+    # XXX: Those only work when installed from the store
```

Comment:
Ha, this is a debug comment. I think it is not really true. Thanks for spotting this.